webSocketFunctions.js

```diff
@@ -18,7 +18,14 @@ const playersInGame = (game) => new Set([...game?.players].map((player) => playe
 const isHost = (gameId, userId) => games[gameId].host === userId;
 
 const assignRoles = (game) => {
-  const roles = ["LEADER", "ASSASSIN", "ASSASSIN", "GUARDIAN", "TRAITOR"];
+  const roleSetups = {
```

Comment:
We'll want to move this to the top of the file. Its typically a convention for static data like this to be at the top of the file, after the imports.

Reply:
Done